packages/clay-css/src/scss/components/_date-picker.scss

```diff
@@ -0,0 +1,103 @@
+.date-picker {
+	.input-group-text {
+		@include clay-input-group-text-variant($date-picker-input-group-text);
```

Comment:
hey @pat270, I think we'll need this customization also for ClayTimePicker outside of DatePicker as well.

Reply:
@matuzalemsteles we can add it when we make ClayTimePicker. I'd like to keep them separate even though Lexicon defines the styles the same.

Reply:
ok, np.